### PR TITLE
[SA-3133] Add UI extensions for new routing

### PR DESCRIPTION
**Pull request by connor-baer**
Addresses [SA-3133](https://sumupteam.atlassian.net/browse/SA-3133).

## Changes

* **Add "Copy Value" extension:** This extension displays the field value in a read-only input as well as a button to copy the field value to the clipboard. You can use this extension with 'Symbol', 'Text', 'Integer', and 'Number' field types.

## Status

* [x] Unit tested
* [x] Accessible
* [x] Documented
* [x] Tested in Contentful

### CODE REVIEW

File: packages/route-url/gulpfile.babel.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import gulp from 'gulp';
```
Comment (felixjung) [inline packages/route-url/gulpfile.babel.js]: So you didn't go with Parcel in the end? 😏

File: packages/route-url/gulpfile.babel.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+const reload = done => {
+  server.reload();
+  done();
+};
+
+export const clean = () => del([DEST]);
+
+const scripts = () =>
+  browserify(PATHS.scripts)
```
Comment (felixjung) [inline packages/route-url/gulpfile.babel.js]: But went with browserify? Interesting 😀

File: packages/route-url/gulpfile.babel.js (hunk 1 above)
Comment (connor-baer) [reply packages/route-url/gulpfile.babel.js]: Extensions have the very specific requirement that we can only upload one file to Contentful. So unless we host the extensions ourselves (which would have been a bigger hassle), I needed to find a way to inline the styles and scripts.
Parcel doesn't support that, but with gulp it's easy to implement.

File: packages/route-url/gulpfile.babel.js (hunk 2 above)
Comment (connor-baer) [reply packages/route-url/gulpfile.babel.js]: I guess webpack would work as well 🤔

**Pull request merged**